TwilioVerifyDemo/TwilioVerifyDemo/Common/AppDelegate.swift

```diff
@@ -82,28 +86,60 @@ extension AppDelegate: UNUserNotificationCenterDelegate {
       return
     }
     
-    showChallenge(payload: payload)
+    handleChallengeApproval(with: payload)
   }
 }
 
 private extension AppDelegate {
+  func handleChallengeApproval(with payload: [String: Any]) {
+    guard let payloadData = PushChallengePayloadData(payload: payload) else {
+      return
```

Comment:
Let's add an error log here, to left breadcrumb in case we were missing any payload info.